DEVELOPING.md

```diff
@@ -15,6 +15,8 @@ helm install --wait \
     rancher-backup ./charts/rancher-backup
 ```
 
+For help with setting up the environment, check out `/scripts/deploy`. It has some helpful scripts for deploying the various components used in the development environment. For more information on the script try running `./scripts/deploy`.
```

Comment:
This needs a bit more explaining to the user how the development workflow should be, so what commands to use in what order to make use of the benefits. And the requirement for DockerHub.

Reply:
True, although I did add a lot of information to the script generic input. But i'll copy it here.

Reply:
Its not about showing the options and configuration, it is about explaining to the user what the flow is when they want to use the functionality. For example;
* make your code changes and commit
* running `make` will build, test, validate and run integration tests on embedded k3s cluster in dapper
* if you want to test your changes in your own cluster, set ENV vars X Y Z, and run `make build`, `make package` and use the `scripts/deploy` helper script
* and then the options and what they do if they want to test their local changes (publish the image to dockerhub, then deploy minio, and deploy backup-restore-operator) etc